src/pages/documentation/index.md

```diff
@@ -2,7 +2,7 @@
 title: Mobile SDK overview
 description: This is the overview page of Mobile SDK
 contributors:
-  - https://github.com/icaraps 
+  - https://github.com/icaraps
```

Comment:
Honestly, I don't think we need to keep this, since you're here anyways 😂 (But this one doesn't matter _that_ much).

Reply:
not sure what happens if I remove it or do we need to add our contributors list to it?

Reply:
<img width="1726" alt="Screenshot 2023-02-22 at 10 45 11 AM" src="https://user-images.githubusercontent.com/74737634/220728320-f40942af-02ab-4d00-b61d-af020ca6db52.png">

I think it's the thing that causes that to appear. Let me confirm what happens if we remove it, though.

Reply:
@emdobrin I've confirmed we can remove the contributors! 

Though, at some point in the future, we should add the metadata (as in the title/description at the very least) to all our pages:

```
---
title: Mobile SDK overview
description: This is the overview page of Mobile SDK
---
```